src/app/screens/Onboard/ConnectGaloy/index.tsx

```diff
@@ -0,0 +1,258 @@
+import { useState } from "react";
+import { useNavigate } from "react-router-dom";
+import axios from "axios";
+import Input from "../../../components/Form/Input";
+import Button from "../../../components/Button";
+
+import utils from "../../../../common/lib/utils";
+
+const url = process.env.GALOY_URL || "https://api.staging.galoy.io/graphql/";
```

Comment:
we should default here to the production URL

Reply:
Done: https://github.com/getAlby/lightning-browser-extension/pull/505/commits/e73f383a5264571709fc1dc4972b337031e295bf